sentry-android-core/src/main/java/io/sentry/android/core/AndroidOptionsInitializer.java

```diff
@@ -155,12 +163,18 @@ private static void installDefaultIntegrations(
           new ActivityLifecycleIntegration(
               (Application) context, buildInfoProvider, activityFramesTracker));
       options.addIntegration(new UserInteractionIntegration((Application) context, loadClass));
+      if (isIntegrationAvailable(SENTRY_FRAGMENT_INTEGRATION_CLASS_NAME, options, loadClass)) {
+        options.addIntegration(new FragmentLifecycleIntegration((Application) context));
```

Comment:
What if `FragmentLifecycleIntegration` is already added?
Does this run before or after the `configuration` callback?

Reply:
Yeah it runs before the config callback - https://github.com/getsentry/sentry-java/blob/626f92f81b9a2c5c71b50fcffe083b6c0c140285/sentry-android-core/src/main/java/io/sentry/android/core/SentryAndroid.java#L74

Reply:
The problem is then if they don't see the changelog/migration, and still add it manually, it will lead to duplicated data, how do we prevent that?

Reply:
Good point! In theory we could add a `sanitize` method, which runs after `configuration.configure(options)` and removes duplicate integrations. We'd need some kind of a hint or something for the integration though, to distinguish between user-added and auto-added one. Any other ideas or this sounds fine?

Reply:
Nope, Some SDKs have a `defaultIntegrations` list instead of using a single one, See https://docs.sentry.io/platforms/javascript/configuration/options/#default-integrations
We don't have that on Java/Android, otherwise, it'd be easy to make a difference.
Worth discussing with @bruno-garcia as well.

Reply:
Do we have any integration that legit can be added twice? If not, simplest approach might be to filter out duplicates by type before calling `register`. It's a change in behavior that even if we don't have any integration that can be duplicated, users could rely on it but being honest, it's very unlikely so I think we could go ahead without bumping a major here just adding a note to the changelog.

If we use the last registered integration, we have a path to allow the auto configure work in tandem with the manual one. So we add automatically (with defaults) but if users want to provide options, they can add manually and we'll use that instance instead.